pkg/backend/display/progress.go

```diff
@@ -1217,7 +1217,8 @@ func (display *ProgressDisplay) processNormalEvent(event engine.Event) {
 		row.SetHideRowIfUnnecessary(false)
 	}
 
-	if event.Type == engine.ResourcePreEvent {
+	switch event.Type { //nolint:exhaustive // golangci-lint v2 upgrade
```

Comment:
Why `// golangci-lint v2 upgrade`? Are these things we should be following up on sometime later?

Reply:
Mostly so we don't later assume that this was intentionally non-exhaustive. This is meant as a "there was no thought put in here" comment.